automation/taskcluster/androidTest/ui-test.sh

```diff
@@ -80,6 +80,8 @@ elif [[ "${device_type}" == "x86-start-test" ]]; then
     flank_template="${PATH_TEST}/flank-x86-start-test.yml"
 elif [[ "${device_type}" == "arm-start-test" ]]; then
     flank_template="${PATH_TEST}/flank-armeabi-v7a-start-test.yml"
+elif [[ "${device_type}" == "x86-screenshots-tests" ]]; then
```

Comment:
⚠️ Screenshots aren't exposed on Taskcluster. This script needs to be updated to download them and move them to `/builds/worker/artifacts/`

Reply:
Actually this task does not get the screenshots, it only runs the tests. What we could expose there is the test results?

Reply:
Cool, then it seems we're all done, I see some test results publihsed https://firefox-ci-tc.services.mozilla.com/tasks/Rbj7yALeQy--ZhDx6kMmcw#artifacts 😃